compiler/src/dotty/tools/dotc/core/JavaNullInterop.scala

```diff
@@ -158,26 +99,31 @@ object JavaNullInterop {
           !tp.isRef(defn.RepeatedParamClass)
         case _ => true
       })
-    }
 
     /** Should we nullify the type arguments to the given generic `tp`?
      *  We only nullify the inside of Scala-defined generics.
      *  This is because Java classes are _all_ nullified, so both `java.util.List[String]` and
      *  `java.util.List[String|Null]` contain nullable elements.
      */
-    def needsNullArgs(tp: AppliedType): Boolean = {
-      !tp.classSymbol.is(JavaDefined)
-    }
+    def needsNullArgs(tp: AppliedType): Boolean = !tp.classSymbol.is(JavaDefined)
 
     override def apply(tp: Type): Type = {
       tp match {
-        case tp: TypeRef if needsTopLevelNull(tp) => tp.toJavaNullableUnion
+        case tp: TypeRef if needsTopLevelNull(tp) =>
```

Comment:
Could you please revert this line, and the ones below where mapOver was moved to its own line?

In general, we should avoid formatting changes because they "pollute" the diff, making it harder to review.